hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/service/NftAllowanceService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.restjava.service;
+
+import com.hedera.mirror.common.domain.entity.NftAllowance;
+import java.util.List;
+
+public interface NftAllowanceService {
+
+    List<NftAllowance> getNftAllowances(NftAllowanceRequest request);
```

Comment:
Prefer higher level interfaces.
```suggestion
    Collection<NftAllowance> getNftAllowances(NftAllowanceRequest request);
```